test/integration/phase_label_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class PhaseLabelTest < ActionDispatch::IntegrationTest
+  test "Beta phase label is displayed for a Service Manual Guide in phase 'beta'" do
+    ENV["FLAG_ENABLE_SERVICE_MANUAL"] = 'yes'
```

Comment:
It would. But it's a temporary feature flag, is it really worth adding a new gem dependency for that?